web/packages/hovercards/src/index.ts

```diff
@@ -1,6 +1,8 @@
 export type { Placement } from './compute-position';
 export type {
 	VerifiedAccount,
+	ContactInfo,
+	Payments,
```

Comment:
We usually export all the possible types to enable developers to handle any cases they might needed:

```suggestion
	ContactInfo,
	Payments,
	PaymentLink,
	CryptoWallet
```